### ISSUE TITLE
kali 4gb didn't boot 

### ISSUE THREAD

**Issue by j0n777** [leeboby/kali-images#15]
-a---- 14/07/2024 21:24 1572864000 kali-linux-2024.2-opizero3-xfce-arm64.img.xzaa
-a---- 14/07/2024 21:44 1138865840 kali-linux-2024.2-opizero3-xfce-arm64.img.xzab
-a---- 12/07/2024 21:58 12567188 win32diskimager-1.0.0-install.exe

PS D:\OrangePi> cat kali-linux-2024.2-opizero3-xfce-arm64.img.xza* > kali-linux-2024.2-opizero3-xfce-arm64.img.xz
PS D:\OrangePi> ls

tried to uncompress this file .xz with 7zip but fail

![image](https://github.com/user-attachments/assets/4752a565-fc0a-40a1-9bc1-a965f4a2e61d)

![image](https://github.com/user-attachments/assets/52726935-1915-4d35-93a6-3753468c9b4a)

**Comment by ilkerzeki** [comment]
Files worked for me without an issue. I used rufus to burn image.
Give a try to rufus and respond for further results...

**Comment by Infinity2282** [comment]
Use the command copy /b "kali-linux-2024.2-opizero3-xfce-arm64.img.xza*" "kali-linux-2024.2-opizero3-xfce-arm64.img.xz" .To merge kali-linux-2024.2-opizero3-xfce-arm64.img.xzaa , kali-linux-2024.2-opizero3-xfce-arm64.img.xzab. And then unpack if it won't work then run the command through CMD.

**Comment by TheSecretComputerNerd** [comment]
Yeah same issue here I just got the minimal install and downloaded packages from there, though that's not to say the minimal isn't without bugs.
